packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -231,7 +191,6 @@ Function AgentCondRestart() {
 }
 
 Function PreunAll() {

```

Comment:
If this is a one-line function, it is wasteful since its does not give any meaning for the function and consider delete the function

Reply:
We can leave it